docs/management/gcache-write-set-cache-encryption.md

```diff
@@ -0,0 +1,100 @@
+# GCache and Write-Set cache encryption
+
+The feature is a [tech preview](../glossary.md#tech-preview). Before using this feature in production, we recommend that you test restoring production from physical backups in your environment, and also use the alternative backup method for redundancy.
+
+## GCache encryption Master Key
+
+GCache encrypts with a File Key. Creating the galera.gcache file or a full reset generates the File Key. The File Key is encrypted with the Master Key. 
```

Comment:
GCache and Write-Set cache files created by Galera are encrypted with File Key if this feature is enabled. Only Ring Buffer file of GCache is non-volatile (it survives the server restart), so only for this file, the File Key has to be retained. It is stored after encrypting with Master Key which is stored in the keyring.
For GCache off-pages and Write-Set cache files, File Key is not stored anywhere.